website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -4,36 +4,38 @@ id: "user-tokens"
 pagination_next: "docs/dbt-cloud-apis/service-tokens"
 ---
```

Comment:
I opted not too because technically it's about a user token type and redirects would be difficult since we want users to find the migration section and redirects to that won't work